server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTreeBelowSGMemoryImpl.java

```diff
@@ -697,7 +754,7 @@ public IMeasurementMNode getMeasurementMNode(PartialPath path) throws MetadataEx
   }
 
   @Override
-  public List<IMeasurementMNode> getAllMeasurementMNode() throws MetadataException {
+  public List<IMeasurementMNode> getAllMeasurementMNode() {
```

Comment:
Is this still a public method used by SchemaRegion? If not, make it private.

Reply:
still a public method used by SchemaRegion..